setup.py

```diff
@@ -24,7 +24,7 @@
 from vsc.install.shared_setup import ag, jt
 
 PACKAGE = {
-    'version': '2.3.1',
+    'version': '2.3.2',
```

Comment:
bump the vsc-config minimum version here as well

Reply:
ok, done.